Sprint-1/implement/max.js

```diff
@@ -1,4 +1,10 @@
 function findMax(elements) {
+    const numbers = elements.filter(element => typeof element === 'number');
+    if (numbers.length === 0) {
+        return -Infinity;
+    }
+    return Math.max(...numbers);
```

Comment:
What do you expect from the following function calls?
Does your function return the value you expected?

```
findMax([-Infinity])
findMax([-Infinity, 0])
findMax(["100", 0])
findMax([0, NaN, 1])
```